ext/pdo_firebird/pdo_firebird.c

```diff
@@ -60,6 +60,7 @@ PHP_MINIT_FUNCTION(pdo_firebird) /* {{{ */
 	REGISTER_PDO_CLASS_CONST_LONG("FB_ATTR_DATE_FORMAT", (zend_long) PDO_FB_ATTR_DATE_FORMAT);
 	REGISTER_PDO_CLASS_CONST_LONG("FB_ATTR_TIME_FORMAT", (zend_long) PDO_FB_ATTR_TIME_FORMAT);
 	REGISTER_PDO_CLASS_CONST_LONG("FB_ATTR_TIMESTAMP_FORMAT", (zend_long) PDO_FB_ATTR_TIMESTAMP_FORMAT);
+	REGISTER_PDO_CLASS_CONST_LONG("FB_ATTR_API_VERSION", (zend_long) PDO_FB_ATTR_API_VERSION);
```

Comment:
Actually, is this related to the server, or the driver?

As it might make more sense to expose it via [`PDO::ATTR_SERVER_INFO`](https://www.php.net/manual/en/pdo.constants.php#pdo.constants.attr-server-info)

Reply:
This only related to the header file `ibase.h` at build time.

The server side is completely irrelevant.

The client side is somewhat involved, but due to factors such as updates, the header file version at the time of build and the client version do not always match.

Reply:
Right, then this is fine.